README.md

```diff
@@ -79,22 +82,36 @@ editor.create(document.createElement('editor'), {
 });
 ```
 
-Also make sure to register the service worker. See the
-[examples](https://github.com/pengx17/monaco-yaml/tree/master/examples) directory for full fledged
-examples.
-
-## Development
+Also make sure to register the service worker.
 
-- `git clone https://github.com/pengx17/monaco-yaml`
-- `cd monaco-yaml`
-- `npm ci`
+## Examples
 
 A running example: ![demo-image](test-demo.png)
 
+Some usage examples can be found in the
+[examples](https://github.com/remcohaszing/monaco-yaml/tree/main/examples) directory.
+
+## Contributing
+
+Please see our [contributing guidelines](CONTRIBUTING.md)
+
 ## Credits
 
-- https://github.com/redhat-developer/yaml-language-server
+Originally [@kpdecker](https://github.com/kpdecker) forked this repository from
+[`monaco-json`](https://github.com/microsoft/monaco-json) by
+[@microsoft](https://github.com/microsoft) and rewrote it to work with
+[`yaml-language-server`](https://github.com/redhat-developer/yaml-language-server) instead. Later
+the repository maintenance was taken over by [@pengx17](https://github.com/pengx17). Eventually the
+repository was tranferred to the account of [@remcohaszing](https://github.com/remcohaszing), who is
+currently maintaining this repository with the help of [fleon](https://github.com/fleon) and
+[@yazaabed](https://github.com/yazaabed).
+
+The heavy processing is done in
```

Comment:
I believe this should not be necessary. Initially the credit was given because `yaml-language-server` source was included in the repo itself, which was later removed and added as a dependency in `package.json`.

Reply:
`yaml-language-server` is still included in the build process. One of the things this project does is make sure `yaml-language-server` is bundled in a way that it works in the browser without the need for additional setup. I can remove it if you have strong feelings about this. It’s already mentioned in the paragraph above.

Reply:
Yes, that's what I meant, its included as a dev dependency, but so are many others. :)

I don't have strong feelings about it, so feel free to merge as is if you like.

Reply:
I’ll merge it as-is then. I appreciate the feedback! Thanks!